ide/static/js/canvas.js

```diff
@@ -237,7 +238,14 @@ class Canvas extends React.Component {
   allowDrop(event) {
     event.preventDefault();
   }
+  clickMultipleLayerEvent(event, layerId) {
+    event.preventDefault();
+    if (event.target.tagName.toLowerCase() == 'div') {
```

Comment:
Isn't there a better way to detect whether the event of click occurred on a layer component?